src_py/sysfont.py

```diff
@@ -95,7 +95,7 @@ def _parse_font_entry_win(name, font, fonts):
     :return: Tuple of (bold, italic, name)
     """
     true_type_suffix = "(TrueType)"
-    mods = ("demibold", "narrow", "light", "unicode", "bt", "mt")
+    mods = ("demibold", "narrow", "light", "unicode", "bt", "mt", "regular")
     if name.endswith(true_type_suffix):
         name = name.rstrip(true_type_suffix).rstrip()
```

Comment:
Unrelated to this PR but I noticed it here

[`rstrip`](https://docs.python.org/3/library/stdtypes.html#str.rstrip) is not the correct method to remove suffixes. This method will strip *any* character that is passed in the argument string and won't take care of the order.

[`removesuffix`](https://docs.python.org/3/library/stdtypes.html#str.removesuffix) is a newly added python string method for doing exactly this, but since we need to maintain 3.6+ compat I suggest
```suggestion
        name = name[:-len(true_type_suffix)].rstrip()
```

Reply:
Yikes, that seems like a really serious bug. I did a blame, it's been like that for at least 6 years.

Reply:
One way to fix this would be to add "(truetype)" to the mods list. That way is uses the already existing mod removal code, so it makes things a little simpler.